lib/include/openamp/virtio.h

```diff
@@ -65,6 +81,7 @@ __deprecated static inline int deprecated_virtio_dev_slave(void)
 struct virtio_device_id {
 	uint32_t device;
 	uint32_t vendor;
+	uint32_t version;
```

Comment:
I don't see this field being used anywhere. Why is it needed?

Reply:
I delete this, thanks.